server/issue_handler_test.go

```diff
@@ -96,6 +96,24 @@ func TestIssueEventHandler(t *testing.T) {
 		require.Equal(t, http.StatusBadRequest, resp.StatusCode)
 	})
 
+	t.Run("Should fail incorrect url", func(t *testing.T) {
```

Comment:
Can we also add a test where repo owner and name is nil but HTMLUrl is set, and verify that `ListLabelsByIssue` is being called with the correct values?